content/knowledge_base/flow/cadence/resources.md

```diff
@@ -0,0 +1,96 @@
+# Resources
+
+Resources are types that can exist only in one location at a time.
+At the end of a function which has resources in scope, resources must either be **moved** or **destroyed**.
+
+After it was _destroyed_, the resource can no longer be used.
+_Moving_ a resource means either assigning it to a different constant or a variable, passing it as an argument to another function, or returning it from a function.
+After the resource was moved - e.g. assigned to a const, the previous reference to the resource is invalid and cannot be used anymore.
+
+To make the resource behaviour clear and explicit, the prefix `@` must be used in all type annotations dealing with resources, and the resource movement is denoted with the _move_ operator - `<-`.
```

Comment:
```suggestion
To make the resource behavior clear and explicit, the prefix `@` must be used in all type annotations dealing with resources, and the resource movement is denoted with the _move_ operator — `<-`.
```

We use `behavior` rather than `behaviour` in our docs since we favor US spelling. I should add a note about it in the markdown style guide, my bad :p